database/seeders/EmailSeriesSeeder.php

```diff
@@ -9,8 +9,8 @@ class EmailSeriesSeeder extends Seeder
 {
     public function run()
     {
-        $series = EmailSeries::where('title', 'Sustaining Fellow Tickets on Sale/Registration Open/RSVP')->first();
-        $series->title = 'Luminary Tickets on Sale/Registration Open/RSVP';
+        $series = EmailSeries::where('title', 'Sustaining Fellow Tickets on Sale/Registration Open/RSVP')->delete();
+        $series = EmailSeries::firstOrNew(['title' =>  'Luminary Tickets on Sale/Registration Open/RSVP']);
```

Comment:
I don't think we want to delete and readd here, because that will break this models relationships to other entities. I think you can instead just get the `first` record and change it, and if there is no existing records just skip the title change.

Reply:
To summarize our convo from yesterday: this change ultimately represents a content change + we're not sure Email Series will continue to be used on the website. Remove for now.